### ISSUE TITLE
Is CBCP (callback control protocol) still useful?

### ISSUE THREAD

**Issue by paulusmack** [ppp-project/ppp#530]
I can't imagine why anyone still needs CBCP, and I'd like to remove it from pppd.  Comment here or email me if you think it should be kept.

**Comment by jkroonza** [comment]
@paulusmack I think perhaps disable by default for the next few releases.  ./configure --enable-cbcp - and if that is triggered, some big warning to come comment on this issue or it will be removed by version X (you can pick).

**Comment by Neustradamus** [comment]
@jkroonza: Yes, the solution to do not remove and to disable by default will be good!

@paulusmack: What do you think?

**Comment by paulusmack** [comment]
It's already disabled by default. I just made a PR which (among other things) prints a warning at configure time if CBCP is enabled. Hopefully that will flush out any actual users.
